torch/_torch_docs.py

```diff
@@ -5897,3 +5900,325 @@
     )
 
 """)
+
+add_docstr(torch.fft,
+           r"""
+fft(input, signal_ndim, normalized=False) -> Tensor
+
+Complex-to-complex Discrete Fourier Transform
+
+This method computes the complex-to-complex discrete Fourier transform.
+Ignoring the batch dimension, it computes the following expression:
+
+.. math::
+    X[\omega_1, \dots, \omega_d] =
+        \frac{1}{\prod_{i=1}^d N_i} \sum_{n_1=0}^{N_1} \dots \sum_{n_d=0}^{N_d} x[n_1, \dots, n_d]
+         e^{-j\ 2 \pi \sum_{i=0}^d \frac{\omega_i n_i}{N_i}},
+
+where :math:`d`=:attr:`signal_ndim` is number of dimensions for the
+signal, and :math:`N_i` is the size of signal dimension :math:`i`.
+
+This method supports 1D, 2D and 3D complex-to-complex transforms, indicated
+by :attr:`signal_ndim`. :attr:`input` must be a tensor with last dimension
+of size 2, representing the real and imaginary components of complex
+numbers. Also :attr:`input` should be of ``signal_ndim + 1`` dimensions or
+``signal_ndim + 2`` dimensions with batched data. If :attr:`normalized` is
+set to ``True``, this returns the normalized Fourier transform results,
+i.e., divided by :math:`\sqrt{\prod_{i=1}^d N_i}`, to become a unitary
+operator.
+
+Returns the real and the imaginary parts together as one tensor of size
+:math:`(* \times 2)`, where :math:`*` is the shape of
+:attr:`input`.
+
+The inverse of this function is :func:`ifft`.
+
+.. warning::
+    For CPU tensors, this method is currently only available with MKL
+    installed.
+
+Arguments:
+    input (Tensor): the input tensor
```

Comment:
What is the format of the dimensions of the input tensor?

Reply:
Specified in the above paragraph.